waffle-doppelganger/contracts/Doppelganger.sol

```diff
@@ -0,0 +1,17 @@
+/* solium-disable security/no-inline-assembly */
+pragma solidity ^0.6.3;
+
+contract Doppelganger {
+    mapping (bytes4 => bytes) mockConfig;
+
+    fallback() external {
+        bytes memory ret = mockConfig[msg.sig];
+        assembly {
+            return(add(ret, 0x20), mload(ret))
+        }
+    }
+
+    function mockReturns(bytes4 key, bytes memory value) public {
```

Comment:
should we add some strange prefix to avoid potential name collision, e.g. `__internal__returns`

Reply:
Why? It's a public method